recipes/texture-atlas/build.sh

```diff
@@ -5,6 +5,16 @@ cd build
 
 BUILD_CONFIG=Release
 
+if [ $(uname) == Darwin ]; then
+    COMP_CC=clang
+    COMP_CXX=clang++
+    export MACOSX_DEPLOYMENT_TARGET="10.9"
```

Comment:
It could be cleaner to use the CMake variable to set this: `CMAKE_OSX_DEPLOYMENT_TARGET`